oak_runtime/src/node/http/mod.rs

```diff
@@ -301,19 +301,35 @@ struct HttpRequestHandler {
 impl HttpRequestHandler {
     async fn handle(&self, req: Request<Body>) -> Result<Response<Body>, OakStatus> {
         let request = HttpServerNode::map_to_http_request(req).await;
-        let oak_label = get_oak_label(&request)?;
-        info!(
-            "Handling HTTP request; request size: {} bytes, label: {:?}",
-            request.body.len(),
-            oak_label
-        );
-
-        debug!("Inject the request into the Oak Node");
-        let response = self
-            .inject_http_request(request, &oak_label)
-            .map_err(|_| OakStatus::ErrInternal)?;
-
-        Ok(response.to_response())
+        match get_oak_label(&request) {
+            Ok(oak_label) => {
+                info!(
+                    "Handling HTTP request; request size: {} bytes, label: {:?}",
```

Comment:
```suggestion
                    "Handling HTTP request; request body size: {} bytes, label: {:?}",
```

I believe the size is that of the body, not of the whole request, right?